test/tests/functional/pbs_reservations.py

```diff
@@ -1124,23 +1124,31 @@ def test_reservations_with_expired_subjobs(self):
         self.server.expect(JOB, {'job_state': 'F', 'substate': '91'},
                            extend='x', id=jid)
 
-    def test_ASAP_resv_with_multinode_job_array(self):
+    def common_config(self):
         """
-        Test multinode job array with ASAP reservation
+        This function contains common steps for test
+        "test_ASAP_resv_with_multinode_job_array" and
+        "test_standing_resv_with_multinode_job_array"
         """
-        if (len(self.moms) < 2):
-            self.skip_test("Test requires 2 moms: use -p mom1:mom2")
-        a = {'resources_available.ncpus': 4}
-        for mom in self.moms.values():
-            self.server.manager(MGR_CMD_SET, NODE, a, id=mom.shortname)
+        vn_attrs = {ATTR_rescavail + '.ncpus': 4}
+        self.server.create_vnodes("vnode1", vn_attrs, 1,
+                                  self.mom, fname="vnodedef1")
+        self.server.create_vnodes("vnode2", vn_attrs, 1,
+                                  self.mom, additive=True, fname="vnodedef2")
```

Comment:
Can you please check if we can create vnode as vn1[0] and vn1[1] instead of vn1[0] and vn2[0]?
If yes, then  delete line 1136 and 1137.